python/hidet/graph/operator.py

```diff
@@ -84,6 +84,8 @@ def build_target(self) -> str:
 
         if isinstance(self, TransferOp):
             return 'cuda'
+        elif self.device.is_vcuda():
+            return 'cuda'
         else:
             return self.device.kind
```

Comment:
```suggestion
        else:
            if self.device.kind in ["cuda", "vcuda"]:
                return "cuda"
            elif self.device.kind == "cpu":
                return "cpu"
            else:
                raise NotImplementedError()
```
Hope this to be more readable.